ui/widgets/nuxeo-user-tag.js

```diff
@@ -158,7 +158,7 @@ import './nuxeo-tooltip.js';
       return this.urlFor('user', this._id(user));
     }
 
-    _hasLink(user) {
+    _hasLink(disabled, user) {
```

Comment:
IMO, the fact that `disabled` value change is not "listened" is a bug and may deserve a dedicated ticket.